src/clusterfuzz/_internal/cron/schedule_fuzz.py

```diff
@@ -186,15 +202,38 @@ def get_batch_regions(batch_config):
   return list(set(config['gce_region'] for config in mapping.values()))
 
 
+def get_available_cpus(project: str, regions: List[str]) -> int:
+  """Returns the available CPUs for fuzz tasks."""
+  # TODO(metzman): This doesn't distinguish between fuzz and non-fuzz
+  # tasks (nor preemptible and non-preemptible CPUs). Fix this.
+  waiting_tasks = sum(
+      batch.count_queued_or_scheduled_tasks(project, region)
+      for region in regions)
+  waiting_tasks += count_unacked(project, 'preprocess')
+  waiting_tasks += count_unacked(project, 'utasks_main')
+  soon_commited_cpus = waiting_tasks * CPUS_PER_FUZZ_JOB
+  logs.info(f'Soon committed CPUs: {soon_commited_cpus}')
+  available_cpus = sum(
+      get_available_cpus_for_region(project, region) for region in regions)
+  available_cpus = max(available_cpus - soon_commited_cpus, 0)
```

Comment:
Now that we take the queue size into account, we can go back to running this very frequently, right?

Reply:
Hmmm...I had the opposite thought, that because we can schedule so many more at once, there's no need to run it so often. I think there can be a slight delay between publishing and reaching the queue, so probably above 5 minutes makes most sense.